contrib/IECoreUSD/test/IECoreUSD/USDSceneTest.py

```diff
@@ -4257,5 +4259,28 @@ def testPerPurposeModelBound( self ) :
 		self.assertTrue( root.child( "group" ).hasBound() )
 		self.assertEqual( root.child( "group" ).readBound( 0 ), imath.Box3d( imath.V3d( -1 ), imath.V3d( 1 ) ) )
 
+	def testAsetPathSlashes ( self ) :
+
+		root = IECoreScene.SceneInterface.create(
+			os.path.join( os.path.dirname( __file__ ), "data", "assetPathAttribute.usda" ),
+			IECore.IndexedIO.OpenMode.Read
+		)
+		xform = root.child( "xform" )
+
+		self.assertEqual( xform.attributeNames(), [ "render:testAsset" ] )
+		self.assertNotIn( "\\", xform.readAttribute( "render:testAsset", 0 ).value )
```

Comment:
Could we strengthen this test  by also asserting that the file exists?

Reply:
Fixed in https://github.com/ImageEngine/cortex/pull/1426/commits/a3c9fff781461093087c019f629421730e65125b